src/wp-includes/class-wp-oembed.php

```diff
@@ -212,6 +210,7 @@ public function __construct() {
 		 * | Photobucket  | photobucket.com      | 2.9.0     | 5.1.0     |
 		 * | Funny or Die | funnyordie.com       | 3.0.0     | 5.1.0     |
 		 * | CollegeHumor | collegehumor.com     | 4.0.0     | 5.3.1     |
+		 * | Hulu         | hulu.com             | 2.9.0     | 5.4.2     |
```

Comment:
Just noting that the deprecated version will need to be adjusted when this makes it in. I am debating including this in 5.5. I'll move that conversation to Trac.